### DISCUSSION TITLE
Error when building with Uno platform on unpacked

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#13544] [Q&A] [answered]
### Current behavior

Hi

I getting this error when trying to do an unpacked build about a 1.3 runtime missing.

Can I please find out the exact version to download there are a few of them.

![Build_Error](https://github.com/unoplatform/uno/assets/12206371/bb1e62e5-e61e-4f68-80dc-d297c26f9454)

i have attached the exact image.

Version numbers I think are different from the 1.3 downloads.

Thanks
Ward

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report!

This is a question for the WinAppSDK/WinUI team, but you can find [the installer here](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads).

**Reply by ward-horsfall** [reply]
Hi jeromelaban,

Awsome thanks for the great response :)

Ward.
